ESSArch_Core/frontend/static/frontend/views/advanced_filters.html

```diff
@@ -0,0 +1,24 @@
+<div>
+  <span
+    class="fas fa-md fa-filter filter-icon"
+    ng-class="{'blue-border': vm.showAdvancedFilters, 'filter-active': vm.filterActive()}"
+    ng-click="vm.toggleAdvancedFilters()"
+    uib-tooltip="{{'FILTERS' | translate}}"
+    tooltip-placement="top"
+    tooltip-append-to-body="true"
+  >
+  </span>
+  <div ng-show="vm.showAdvancedFilters" class="advanced-filters">
+    <form name="form1" style="position: relative;" novalidate>
```

Comment:
How is the `name` attribute used? Do we need it?

Reply:
It's not. We don't. I'll remove it